### PR TITLE
luminous: Revert pg log limit changes

### PR DESCRIPTION
**Pull request by neha-ojha**
Fixes: http://tracker.ceph.com/issues/36686
Signed-off-by: Neha Ojha <nojha@redhat.com>

- [ ] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by yuriw** [comment]
[wip-yuri3-testing-2018-11-03-0009-luminous](https://trello.com/c/PmIarO9c/464-wip-yuri3-testing-2018-11-03-0009-luminous)

**Comment by smithfarm** [comment]
@neha-ojha Could you write up a release note for this in `PendingReleaseNotes`?

File: PendingReleaseNotes [hunk 1]
```diff
@@ -111,3 +111,14 @@
...
+12.2.9
+------
+* 12.2.9 contains the pg hard hard limit patches. A partial upgrade during
+  recovery/backfill, can cause the osds on the previous version, to fail with
+  assert(trim_to <= info.last_complete). The workaround for users is to upgrade
+  and restart all OSDs to a version with the pg hard limit, or only upgrade
+  when all PGs are active+clean. This patch will be reverted in 12.2.10, until
+  a clean upgrade path is added to the pg log hard limit patches.
+
+  See also: http://tracker.ceph.com/issues/36686
```
Comment (smithfarm) [inline PendingReleaseNotes]: Thanks, @neha-ojha, this looks good. The only thing that would make it even better (for me) is to include an explanation of what the "pg hard limit" feature is? If there is a ceph.com blog post then maybe we could just link to it.

Comment (neha-ojha) [reply PendingReleaseNotes]: @smithfarm added the tracker reference that explains this feature.

**Pull request merged**